packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -245,13 +246,19 @@ const Autocomplete = <
   }, [inputValue]);
 
   const renderInput = useCallback(
-    ({ InputLabelProps, InputProps, ...params }) => (
+    ({
+      InputLabelProps,
+      InputProps,
+      ...params
+    }: AutocompleteRenderInputParams) => (
       <Field
         ariaDescribedBy={ariaDescribedBy}
         errorMessage={errorMessage}
         errorMessageList={errorMessageList}
         fieldType="single"
         hasVisibleLabel
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        //@ts-ignore
         id={InputLabelProps.htmlFor}
```

Comment:
Id' like to know why we're `@ts-ignore`ing this. Can you add a comment after the `@ts-ignore`?
```suggestion
        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
        // @ts-ignore Blah is required for bleh. Something caused this because X, Y, Z. @see LINK
        id={InputLabelProps.htmlFor}
```

Reply:
Same for the other ones too.

Reply:
Did this for now as `htmlFor` isn't available on `InputLabelProps` and I didn't have time yet to figure it out.